Image Viewing GUI/README.md

```diff
@@ -0,0 +1,18 @@
+# Image Viewing GUI
+
+## Description:
+
+This is a GUI based script, which will display images from the image folder one by one, as in the case of any image viewing software.
+There is a previous button and a next button in the GUI. When the next button is pressed, the next image in the image folder will be displayed.
+When the previous button is pressed, the previous image will be displayed.
+
+## Modules Used:
+

```

Comment:
Remove the extra spaces here. Also, in modules used, you should only mention the name of the module and not the import statement. Like:

random
pandas
numpy